scripts/events.py

```diff
@@ -412,9 +412,10 @@ def check_achievements(self):
                     achievements.add("3")
                 if num_victims >= 50:
                     achievements.add("4")
-                ##Doesn't currently work, for some reason. - Chibi
```

Comment:
if "is_murderer" is in the cat's murder_history then they have murdered at least one cat

Reply:
That makes sense! I've figured it out, I was misunderstanding the code I was reading. Thank you!